### ISSUE TITLE
MinIO Chart 7.1.1 is broken and not in sync with branch "master"

### ISSUE THREAD

**Issue by aureq** [minio/charts#35]
Hi,

I've been trying to upgrade minio to the current chart `version: 7.1.1` (according to `Chart.yaml`) but this failed.
```bash
helm upgrade --install    --namespace minio    -f values.yaml    minio    minio/minio --dry-run
Error: UPGRADE FAILED: parse error at (minio/templates/deployment.yaml:236): unexpected {{end}}
```

To reproduce the error:
```bash
wget https://github.com/minio/charts/raw/76372e091dd632beb2eab2617f0be5b55548d518/releases/minio-7.1.1.tgz
sha256sum -b minio-7.1.1.tgz
# 22b0421ea4fbd42d6b151f83f5d2f2cc7c4af8f169ef419e4809794a36d97602 *minio-7.1.1.tgz
tar -zxf minio-7.1.1.tgz
cd minio
helm lint .
# ==> Linting .
# [ERROR] templates/: parse error at (minio/templates/deployment.yaml:236): unexpected {{end}}
# 
# Error: 1 chart(s) linted, 1 chart(s) failed
```
The same can be reproduced with the URL: https://helm.min.io/releases/minio-7.1.1.tgz

It appears that `templates/deployment.yaml` is not in sync with the `master` branch for some reason.

```diff
--- templates/deployment.yaml   2020-10-03 17:21:40.233447610 +1000
+++ /tmp/minio-git-master/templates/deployment.yaml    2020-10-03 17:15:47.464988725 +1000
@@ -95,9 +95,8 @@
           {{- end }}
           {{- end }}
           {{- end }}
-          {{- end }}
           volumeMounts:
-            {{- if and .Values.persistence.enabled (not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) (not .Values.b2gateway.enabled) }}
+            {{- if and .Values.persistence.enabled (not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) }}
             - name: export
               mountPath: {{ .Values.mountPath }}
               {{- if .Values.persistence.subPath }}
@@ -209,7 +208,7 @@
 {{ toYaml . | indent 8 }}
 {{- end }}
       volumes:
-        {{- if and (not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) (not .Values.b2gateway.enabled) }}
+        {{- if and (not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) }}
         - name: export
         {{- if .Values.persistence.enabled }}
           persistentVolumeClaim:
```

@harshavardhana Part of the way you (the MinIO team) publish, may I suggest running `helm lint .` as part of an automated CI/CD to avoid syntax breakage please?

Thanks

**Comment by harshavardhana** [comment]
Yes I did helm lint, looks like something odd happened locally.

Let me fix it right away

**Comment by harshavardhana** [comment]
fixed in 7.1.2
